app/scaffold-controllers/navigationController.js

```diff
@@ -31,11 +42,14 @@ function(
         errorController,
         drawerEventHandler
     ) {
+        var layoutPromise = AnchoredLayoutPlugin.init();
```

Comment:
Yes it is necessary to have it in it's own variable. We use it in the `Promise.join` but also pass it into `SearchBarController.init`. The `SearchBarController` inserts the searchBar into the Anchored Layout.